src/Microsoft.ML.LightGbm.StaticPipe/LightGbmStaticExtensions.cs

```diff
@@ -313,7 +313,7 @@ public static (Vector<float> score, Key<uint, TVal> predictedLabel)
             int? minimumExampleCountPerLeaf = null,
             double? learningRate = null,
             int numberOfIterations = Defaults.NumberOfIterations,
-            Action<OneVersusAllModelParameters> onFit = null)
+            Action<OneVersusAllModelParameters<CalibratedModelParametersBase<LightGbmBinaryModelParameters, PlattCalibrator>>> onFit = null)
```

Comment:
Why does this have to be a `PlattCalibrator`?

Reply:
Same question in the LightGbmMulticlassTrainer.cs file.

---
In reply to: [299495459](https://github.com/dotnet/machinelearning/pull/3928#discussion_r299495459) [](ancestors = 299495459)

Reply:
So the `LightGbm` stuff was already hard coded in master in the functions themselves. If you look on line 197 in the master branch in `LightGbmMulticlassTrainer` it is hard coded there to return `FeatureWeightsCalibratedModelParameters<LightGbmBinaryModelParameters, PlattCalibrator>`. So my change just puts that in the function signatures as well. From what I saw, with `LightGbm` you do not have the option to pass in a calibrator, and have to take this default option.